src/gtfs_structures.rs

```diff
@@ -49,8 +56,8 @@ pub struct CalendarDate {
 pub struct Stop {
     #[serde(rename = "stop_id")] pub id: String,
     pub stop_name: String,
-    location_type: Option<u8>,
-    parent_station: Option<String>,
+    #[serde(deserialize_with = "deserialize_location_type")] pub location_type: LocationType,
```

Comment:
rustfmt let you write this on the same line ?

Reply:
It actually puts it in one line